common/src/main/java/net/dodogang/plume/block/TallerPlantBlock.java

```diff
@@ -26,22 +26,26 @@
  */
 @SuppressWarnings("unused")
 public class TallerPlantBlock extends PlantBlock {
-    public static final EnumProperty<TripleBlockPart> PART = EnumProperty.of("triple_block_part", TripleBlockPart.class);
+    public static final EnumProperty<TripleBlockPart> PART =
+        EnumProperty.of("triple_block_part", TripleBlockPart.class);
 
     public TallerPlantBlock(AbstractBlock.Settings settings) {
         super(settings);
         this.setDefaultState(this.stateManager.getDefaultState().with(PART, TripleBlockPart.LOWER));
     }
 
     @Override
-    public BlockState getStateForNeighborUpdate(BlockState state, Direction direction, BlockState newState, WorldAccess world, BlockPos pos, BlockPos posFrom) {
+    public BlockState getStateForNeighborUpdate(
+        BlockState state, Direction direction, BlockState newState, WorldAccess world, BlockPos pos, BlockPos posFrom) {
         TripleBlockPart part = state.get(PART);
         if (direction == Direction.UP && part != TripleBlockPart.UPPER) {
             Block block = newState.getBlock();
-            if (block != this) return Blocks.AIR.getDefaultState();
+            if (block != this)
+                return Blocks.AIR.getDefaultState();
```

Comment:
This is an config option.

Reply:
imo this should have curly brackets around it